ui/admin/app/routes/scopes/scope/storage-buckets/new.js

```diff
@@ -71,7 +79,6 @@ export default class ScopesScopeStorageBucketsNewRoute extends Route {
       region,
       worker_filter,
     });
-    record.scopeModel = scopeModel;
```

Comment:
non-blocking comment: since this is removed, now if we save without choosing a scope and navigate to storage-buckets table list view, the page errors out (using mirage). I'm assuming this won't be an issue when running against a boundary instance since API will probably not let user save if they haven't chosen a scope.

Reply:
@lisbet-alvarez 

Very good point. You are totally correct, under a real usage case, the user should NOT be able to create a storage-bucket without selecting a scope.

But under mirage, nothing stops you from creating a storage-bucket without scope because there is no validation.

I think this is worth to discussion and I will bring it for the next tech time.